libraries/chain/include/eosio/chain/block_log.hpp

```diff
@@ -95,7 +95,7 @@ namespace eosio { namespace chain {
 
          static bool is_pruned_log(const fc::path& data_dir);
 
-         static bool extract_block_range(const fc::path& block_dir, const fc::path&output_dir, block_num_type& start, block_num_type& end, bool rename_input=false);
+         static void extract_block_range(const fc::path& block_dir, const fc::path&output_dir, block_num_type start, block_num_type end);
```

Comment:
`end` normal means `last+1`, I think switcheing `end` to `last` here would improve clarity.
```suggestion
         static void extract_block_range(const fc::path& block_dir, const fc::path&output_dir, block_num_type start, block_num_type last);
```